documentation/docs/pipelines/abapEnvironment/configuration.md

```diff
@@ -32,24 +32,7 @@ The annotation `@Library('piper-lib-os')` is a reference to the Jenkins Configur
 
 An Overview of the releases of the Piper Library can be found [here](https://github.com/SAP/jenkins-library/releases).
 
-## 3. Manifest for Service Creation
-
-Create a file `manifest.yml`. The pipeline will create a SAP Cloud Platform ABAP Environment System in the beginning (and delete it in the end). This file describes the ABAP instance, which will be created:
-
-```yaml
----
-create-services:
-- name:   "abapEnvironmentPipeline"
-  broker: "abap"
-  plan:   "standard"
-  parameters: "{ \"admin_email\" : \"user@example.com\", \"description\" : \"System for ABAP Pipeline\", \"is_development_allowed\" : true, \"sapsystemname\" : \"H02\", \"size_of_persistence\" : 4, \"size_of_runtime\" : 1 }"
-```
-
-The example values are a suggestion. Please change them accordingly and don't forget to enter your own email address. Please be aware that creating a SAP Cloud ABAP Environment instance may incur costs.
-
-Please have a look at the [step documentation](https://sap.github.io/jenkins-library/steps/cloudFoundryCreateService/) for more details.
-
-## 4. Configuration for the Communication
+## 3. Configuration for the Communication
 
 The communication to the ABAP system is done using a Communication Arrangement. The Communication Arrangement is created during the pipeline via the command `cf create-service-key`. The configuration for the command needs to be stored in a JSON file. Create the file `sap_com_0510.json` in the repository with the following content:
```

Comment:
Maybe you could additionally point out that this Service Key will be used for the follow-up system provisioning process in the pipeline after the system has been created.